cypress/integration/data-import/view-all-filter.spec.js

```diff
@@ -0,0 +1,137 @@
+import DataImportViewAllPage from '../../support/fragments/data_import/dataImportViewAllPage';
+import fileManager from '../../support/utils/fileManager';
+import getRandomPostfix from '../../support/utils/stringTools';
+
+describe('ui-data-import: Filter the "View all" log screen', () => {
+  // Create unique name for a MARC file
+  const uniqueFileName = `test${getRandomPostfix()}.mrc`;
+  let query;
+
+  before(() => {
+    cy.login(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+
+    cy.getToken(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+
+    // create dynamically file with given name in fixtures
+    fileManager.createFile(`cypress/fixtures/${uniqueFileName}`);
+
+    // remove generated test file from fixtures after uploading
+    cy.uploadFile(uniqueFileName);
+    fileManager.deleteFile(`cypress/fixtures/${uniqueFileName}`);
+  });
+
+  it('C11113 Filter the "View all" log screen', () => {
+    DataImportViewAllPage.gotoViewAllPage();
+
+    // FILTER By "Errors in Import"
+    DataImportViewAllPage.errorsInImportFilters.forEach((filter) => {
+      DataImportViewAllPage.filterJobsByErrors(filter);
+
+      // fetch matched jobs from server
+      query = DataImportViewAllPage.getErrorsQuery({ filter });
+      DataImportViewAllPage.getNumberOfMatchedJobs({ query }).then(count => {
+        DataImportViewAllPage.checkRowsCount(count);
+
+        DataImportViewAllPage.resetAllFilters();
+        DataImportViewAllPage.sortByCompletedDateInDescendingOrder();
```

Comment:
Additionally, do you really need this step into this place?

Reply:
Yes, because when we reset all filters, it clears out sorting filter (e.g. ascending/descending). If we doesn't sort before filtering, server can not parse request params and returns `422` error status.

Reply:
let's wrap to reset method and add your comment about 422 into this new fragment method too